### PR TITLE
Add iOS target

### PR DESCRIPTION
**Pull request by robb**
I also renamed the old target to Set-Mac

### CODE REVIEW

File: Set.xcodeproj/xcshareddata/xcschemes/Set-Mac.xcscheme [hunk 1]
```diff
@@ -15,22 +15,22 @@
...
             </BuildableReference>
          </BuildActionEntry>
          <BuildActionEntry
             buildForTesting = "YES"
-            buildForRunning = "NO"
+            buildForRunning = "YES"
             buildForProfiling = "NO"
             buildForArchiving = "NO"
-            buildForAnalyzing = "NO">
+            buildForAnalyzing = "YES">
```
Comment (robrix) [inline Set.xcodeproj/xcshareddata/xcschemes/Set-Mac.xcscheme]: Could you please disable building the tests for running & analyzing for both Set-Mac & Set-iOS? Xcode enables them by default but then sometimes refuses to build via `xcodebuild` until they’re disabled :confused:

**Comment by robrix** [comment]
I’m not sure how to convince Travis to work for iOS as well. Other than the CI stuff, just the one note—thanks very much @robb! :sparkles:

**Comment by robb** [comment]
I'm at loss what's up with travis, too. It works fine for @robb/Cartography and I'm not sure how their configuration differs. I'll just stop running the iOS specs for now.

**Comment by robb** [comment]
And we're green

**Comment by robb** [comment]
Also, would you mind cutting 1.3 when this is through? :sparkling_heart:

**Comment by robb** [comment]
Let's see if the tests go through if the deployment target is lowered to 8.0

**Comment by robb** [comment]
It worked :green_heart:

**Comment by robrix** [comment]
:sparkles: thank you! I’ll definitely tag 1.3 too.

**Pull request merged**